docs/settings.md

```diff
@@ -0,0 +1,28 @@
+# Module Settings
+
+## Overview
+
+Platform settings are collected and delivered to Settings module by all platform modules.
+Each module can add settings to a standard UI by declaring them in the module manifest.
```

Comment:
In VC3 settings declaring moved to ModuleConstants.cs 
Add an example of how to declare settings

```C#
 public static SettingDescriptor ModuleNameEnabled { get; } = new SettingDescriptor
                {
                    Name = "ModuleName.ModuleNameEnabled",
                    GroupName = "ModuleName|General",
                    ValueType = SettingValueType.Boolean,
                    DefaultValue = false
                };

```

Reply:
Added! Not sure the format is correct in the document, plz check